app/controllers/collections_controller.rb

```diff
@@ -46,6 +46,19 @@ def use
     render :json => result, status: :ok
   end
 
+  def collection_relevancy
+    number_of_collections = 20
+
+    data = {
+      query: params[:query],
+      collections: params[:collections][0..number_of_collections - 1],
```

Comment:
It looks like that we only return the first 20 collections on purpose. Do we care about which collection the user clicked if it is the 32nd collection?